00-projects-template/Makefile

```diff
@@ -4,6 +4,7 @@
 LIBRARY = 	DS
 TESTERS	=	dll que stk set
 RESULTS	=	results
+INCLUDE	=	-I ../Libft
```

Comment:
Including `-I` here could be problematic, check the `all` rule which uses the `ar` command.